projects/admin/src/app/circulation/items.service.ts

```diff
@@ -143,4 +143,37 @@ export class ItemsService {
       })
       );
   }
+
+  /**
+   * Get the available pickup locations for an item.
+   * @param itemPid: the item pid to be requested
+   */
+  getPickupLocations(itemPid) {
```

Comment:
missing return type of the method + @return in doc block

Reply:
done